aws_benchmark/driver.sh

```diff
@@ -24,6 +24,8 @@ program_required terraform
 program_required virtualenv
 program_required python3
 program_required pip
+program_required aws
+program_required ansible-playbook
```

Comment:
this should be installed by the `pip install "ansible==5.0.1"` line in `bring_up_resources.sh`, not required to run driver.sh

Reply:
specifically I mean ansible-playbook, not aws :)

Reply:
done in cf0b91a